extensions/wikia/PageHeader/PageHeader.setup.php

```diff
@@ -21,3 +21,24 @@
 
 // i18n
 $wgExtensionMessagesFiles[ 'PageHeader' ] = $dir . 'PageHeader.i18n.php';
+
+$wgResourceModules['ext.wikia.pageHeader.styles'] = [
+	'styles' => [ 'styles/index.scss' ],
+
+	// use domain sharding
+	'source' => 'common',
```

Comment:
Page Header styles use Theme Designer values, so we shouldn't cache the final CSS between wikis. Is that what this line does?

Reply:
It just makes sure we use `nocookie.net` domain to load this CSS - like AM does. The Sass variables still get passed via the URL. However maybe we could just use the local wiki domain.